libsrc4/nc4dispatch.c

```diff
@@ -59,6 +59,7 @@ NC4_inq_var_all,
 NC4_var_par_access,
 NC4_def_var_fill,
 
+#ifdef USE_NETCDF4
```

Comment:
@mathstuf I'm wondering if there is a reason for this USE_NETCDF4 clause. Since the libsrc4 directory will not be built unless USE_NETCDF4 is true, I don't believe it is necessary. Also, seemingly NC4_def_var_fill, and the other functions above this would also not be defined if USE_NETCDF4 is not defined. So put this ifdef around these specific functions? Just curious...

Reply:
Note that this was done in the course of fitting the netcdf build to be embedded inside of VTK, so maybe I mixed up some settings, but I was getting warnings about this code having extra initial elements compared to the struct declaration.

Looking at the CMake code, `libsrc4` is guarded by `if(USE_HDF5)`, not on `if(USE_NETCDF4)`.